common/lib/request.js

```diff
@@ -1,8 +1,8 @@
 const queryString = require('query-string')
 
+const getQueryString = (target, source) =>
+  `?${queryString.stringify({ ...target, ...source })}`
+
 module.exports = {
-  getQueryString(target, source) {
-    const merged = Object.assign({}, target, source)
-    return `?${queryString.stringify(merged)}`
-  },
+  getQueryString,
```

Comment:
I quite like this shorthand style when there's only 1/2 methods being exported, but that's personal preference I suppose. I'll let you decide which is best :)

Reply:
> I quite like this shorthand style when there's only 1/2 methods being exported, but that's personal preference I suppose. I'll let you decide which is best :)

I hear yah - changed to a standard function 🙌